src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color,double side) {
```

Comment:
There is a missing space after the comma in the constructor parameter list `String color,double side`. It should be `String color, double side`.